benchmark-service/benchmark/benchmark.py

```diff
@@ -232,17 +235,22 @@ async def update_commits(app):
     for gh_commit in gh_data:
         sha = gh_commit.get('sha')
         log.info(f'for commit {sha}')
-        file_path = f'gs://{HAIL_BENCHMARK_BUCKET_NAME}/benchmark-test/{sha}.json'
+        file_path = f'{BENCHMARK_RESULTS_PATH}/{sha}.json'
         has_results_file = gs_reader.file_exists(file_path)
         batches = [b async for b in batch_client.list_batches(q=f'sha={sha} running')]
```

Comment:
Can we move this line to the `elif batches` below. We shouldn't check the running batches until we need to.